app/scripts/lib/auth-errors.js

```diff
@@ -190,10 +190,6 @@ function (_, Errors) {
       errno: 1021,
       message: UNEXPECTED_ERROR_MESSAGE
     },
-    ILLEGAL_IFRAME_PARENT: {
-      errno: 1022,
-      message: t('Firefox Accounts can only be placed into an IFRAME on approved sites')
-    },
```

Comment:
I was wondering how to handle error removal. I could leave the error in place and change the `message` to something like `deprecated` so that it's understood to no longer be used.